snapcraft/plugins/rust.py

```diff
@@ -93,6 +91,9 @@ def __init__(self, name, options, project):
 
     def build(self):
         super().build()
+
+        self._write_config()
```

Comment:
This is only for cross-compiling right? What do you think about naming it _write_crosscompile_config?